.ci/scripts/conformance_test.sh

```diff
@@ -27,15 +27,16 @@ source ${CONDA_DIR}/etc/profile.d/conda.sh
 export PATH=${CONDA_DIR}/bin:$PATH
 conda create -y -n conf python=3.7
 source activate conf
-conda install -y -c intel mpich tbb-devel numpy pytest scikit-learn pandas
+conda install -y -c intel mpich tbb-devel numpy pytest pandas
```

Comment:
Per my understanding tbb is not requered for daal4py now.

Reply:
Thanks, fixed